woodpecker/ember/app/.test-pr.yml

```diff
@@ -22,6 +27,5 @@ steps:
     image: danlynn/ember-cli:4.12.1
     commands:
       - npm run test:ember
-    failure: ignore
```

Comment:
I think we should only recommend this option when onboarding a project (with failing tests).